examples/chat/tui/actions.nim

```diff
@@ -93,6 +96,25 @@ proc action*(self: ChatTUI, event: InputString) {.async, gcsafe, nimcall.} =
 
     self.printInput(input)
 
+# AddWalletAccountResult ----------------------------------------------------------
+
+proc action*(self: ChatTUI, event: AddWalletAccountResult) {.async, gcsafe,
+  nimcall.} =
+
+  # if TUI is not ready for output then ignore it
+  if self.outputReady:
+    if event.error != "":
+      self.wprintFormatError(event.timestamp, event.error)
+      return
+
+    let
+      timestamp = event.timestamp
+      name = event.name
+      address = $event.address
+
+    self.printResult("Added wallet account:", timestamp)
+    self.printResult(fmt"{2.indent()}{name} ({address})", timestamp)
```

Comment:
I kind of feel like there should be a way to see the entire address so that a user can double-check the address before executing any transactions. What I have done is show the abbreviated version upon wallet creation, but kept the long form for listing.